openedx/core/djangoapps/content/learning_sequences/api/processors/special_exams.py

```diff
@@ -0,0 +1,104 @@
+"""
+Author: @pcockwell
+Date: Oct 10, 2020
+
+During the review phase for this processor, @giovannicimolin
+discovered what is likely undesired behavior that exists in this
+processor's implementation and in the implementation of the
+`MilestonesAndSpecialExamsTransformer`.
+
+As currently designed, this processor ignores the course specific
+`Enable Timed Exams` setting when determining whether or not it should
+remove keys and/or supplement exam data. It is not entirely clear if
+the behavior should be modified, so it has been decided to consider any
+necessary fixes in a new ticket.
+
+Please see the PR and discussion linked below for further context
+https://github.com/edx/edx-platform/pull/24545#discussion_r501738511
+"""
```

Comment:
@giovannicimolin 

Is this sufficient for a the requested comment? It's been there all along

Reply:
@pcockwell Can you just remove the mention from the code? 

Change it to something like this:
```
As currently designed, this processor ignores the course specific
`Enable Timed Exams` setting when determining whether or not it should
remove keys and/or supplement exam data. This matches the exact behavior
of `MilestonesAndSpecialExamsTransformer`. It is not entirely clear if
the behavior should be modified, so it has been decided to consider any
necessary fixes in a new ticket.

Please see the PR and discussion linked below for further context
https://github.com/edx/edx-platform/pull/24545#discussion_r501738511
```